### PR TITLE
Approved content update

### PR DESCRIPTION
**Pull request by kindavishal**
Updating with new content approved till date.

### CODE REVIEW

File: config/authors/aseemwangoo.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "aseemwangoo",
+  "bio": "",
```
Comment (5wathi) [inline config/authors/aseemwangoo.json:3]: Founder of Flatteredwithflutter. YouTuber, blogger, event speaker and technical writer

File: config/authors/manojbhadane.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Manoj",
+  "bio": "Experienced software engineer with demonstrated history of working in telecommunications industry along with many other sectors like education, e-commerce etc.",
```
Comment (5wathi) [inline config/authors/manojbhadane.json:3]: Android Developer & UI/UX Ninja

File: config/google-maps/repos/joreilly-GalwayBus.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  ],
+  "owner": "joreilly",
+  "repo": "GalwayBus",
+  "name": "GalwayBus",
+  "shortDescription": "Kotlin Multiplatform bus information related project using Jetpack Compose and SwiftUI.  Includes use of Maps Compose for showing maps with bus stops and realtime bus positions.",
+  "longDescription": "Kotlin Multiplatform bus information related project using Jetpack Compose and SwiftUI.  Includes use of Maps Compose for showing maps with bus stops and realtime bus positions.",
+  "content": "README.md",
+  "pages": [],
+  "tags": [
+    "maps-sdk-android"
```
Comment (5wathi) [inline config/google-maps/repos/joreilly-GalwayBus.json:16]: Additional Tags: Kotlin, Jetpack Compose

**Review by 5wathi — CHANGES_REQUESTED** [review]

File: config/google-maps/repos/joreilly-GalwayBus.json (hunk 3 above)
Comment (kindavishal) [reply config/google-maps/repos/joreilly-GalwayBus.json:16]: Hi Swathi,

Jetpack Compose is not a valid tag in our repo.
You can take a look at valid tags [here](https://github.com/google/devlibrary/blob/c016e34c75ffa1bd07067be3582f28ace301649d/shared/product.ts#L296C1-L331)

Comment (kindavishal) [reply config/google-maps/repos/joreilly-GalwayBus.json:16]: We do have Compose, I've added that

**Comment by 5wathi** [comment]
No notes. LGTM @kindavishal

**Pull request merged**